doc/release.md

```diff
@@ -43,7 +70,62 @@ git config branch.master.remote origin
 git config branch.master.merge refs/heads/master
 ```
 
-To announce the release on [ls.implicit.ly](http://ls.implicit.ly/), use
-[Herald](https://github.com/n8han/herald#install) after adding release notes in
-the `notes/` dir.  Also regenerate the catalog with `lsWriteVersion` SBT task
-and `lsync`, in project job-server.
+#### Only one Scala version was published
+
+1. Reset your _local_ branch back to the last commit before the release.
+    ```
+    git reset --hard HEAD^^
+    ```
+2. Set Scala version to the Scala version, which was not published:
+    ```
+    export SCALA_VERSION=${NOT_PUBLISHED_VERSION}
```

Comment:
BTW another shortcut in SBT is to use `+<scala_version>`, or it could be `++`....  then no need to exit the shell.